workshop/docs/modules/ROOT/pages/deploying-a-model-single-model-server.adoc

```diff
@@ -17,20 +18,20 @@ NOTE: Depending on how model serving has been configured on your cluster, you mi
 
 . In the *Single-model serving platform* tile, click *Deploy model*.
 . In the form, provide the following values:
-.. For *Model Name*, type `fraud`.
+.. For *Model deployment name*, type `fraud`.
 .. For *Serving runtime*, select `OpenVINO Model Server`.
-.. For *Model framework*, select `onnx-1`.
+.. For *Model framework (name - version)*, select `onnx-1`.
 .. For *Existing data connection*, select `My Storage`.
 .. Type the path that leads to the version folder that contains your model file: `models/fraud`
 .. Leave the other fields with the default settings.
 +
-image::model-serving/deploy-model-form-sm.png[Deploy model form for single-model serving, 500]
+image::model-serving/deploy-model-form-sm.png[Deploy model from for single-model serving, 500]
```

Comment:
? it should be "form" not "from" 
The screen capture shows a form

Reply:
Ah, my bad - will fix